src/script/vco-ie/main_warhammer/wh2_dlc13_emp_the_huntmarshals_expedition/missions.lua

```diff
@@ -80,8 +80,7 @@ local missions = {
 				objective
 				{
 					override_text mission_text_text_vco_emp_the_2_the_huntsmarshals_expedition_artifacts;
-					type RAZE_OR_SACK_N_DIFFERENT_SETTLEMENTS_INCLUDING;
-					total 2;
+					type RAZE_OR_OWN_SETTLEMENTS;
```

Comment:
No `total`? Does it accept a `total`? Just to write proper docs.

Reply:
Doesn't work. @OhManTFE tested this, is has to be removed. https://discord.com/channels/1005385230552670339/1021682533911511062/1030540842387132547